code/modules/overmap/ships/simulated.dm

```diff
@@ -323,6 +326,22 @@
 	for(var/area/shuttle_area as anything in shuttle.shuttle_areas)
 		shuttle_area.rename_area("[new_name] [initial(shuttle_area.name)]")
 	return TRUE
+/obj/structure/overmap/ship/simulated/proc/set_ship_faction()
+	if(!COOLDOWN_FINISHED(src, faction_cooldown))
+		return
+	COOLDOWN_START(src, faction_cooldown, 5 MINUTES)
```

Comment:
turn this into a define (`#define FACTION_CHANGE_COOLDOWN 5 MINUTES` at the top of the file, `#undef FACTION_CHANGE_COOLDOWN` at the end, also bump it to 20 minutes at jackrip's request